UPGRADE.md

```diff
@@ -14,7 +14,7 @@
 
 ### ApiBundle
 
-* All api routes are prefixed with version number. For example route: `/api/customers` has changed to `/api/v1/customers`. 
+* Change the import path of your API routing from `src/Sylius/Bundle/ApiBundle/Resources/config/routing/main.yml` to `src/Sylius/Bundle/ApiBundle/Resources/config/routing.yml`. From now one you need to add a version prefix to your API calls(for example route: `/api/customers` has changed to `/api/v1/customers`). 
```

Comment:
`From now one you need to add a version prefix to your API calls(for example route: `/api/customers` has changed to `/api/v1/customers`).`

->

```API became versioned, so you need to prefix them accordingly (e.g. `/api/customer` -> `/api/v1/customer`).```

Wdyt?